htdocs/class/xoopsform/formtextdateselect.php

```diff
@@ -153,7 +153,11 @@ function showCalendar(id)
                 ');
             }
         }
-
-        return "<input type='text' name='" . $ele_name . "' id='" . $ele_name . "' size='" . $this->getSize() . "' maxlength='" . $this->getMaxlength() . "' value='" . $display_value . "'" . $this->getExtra() . " /><input type='reset' value=' ... ' onclick='return showCalendar(\"" . $ele_name . "\");'>";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){            
```

Comment:
__[code_sniffer]__
* Expected 1 space after closing parenthesis; found 0
* Whitespace found at end of line

<!-- Comment by [SideCI](https://sideci.com) -->